paper-dialog-behavior.html

```diff
@@ -128,11 +137,34 @@
       } else {
         this.setAttribute('aria-modal', 'false');
       }
-      // modal implies noCancelOnOutsideClick and withBackdrop if true, don't overwrite
-      // those properties otherwise.
+
+      // modal implies noCancelOnOutsideClick, noCancelOnEscKey and withBackdrop.
+      // We need to wait for the element to be ready before we can read the
+      // properties values.
+      if (!this.__ready) {
```

Comment:
I think there is a `Polymer.Base` property for this. `this._readied` should be true by the time that `ready` is called. Can you try using that instead of creating a new property?